syncstorage-postgres/syncstorage_postgres_db.md

```diff
@@ -14,10 +14,10 @@ Stores per-user, per-collection metadata.
 
 | Column          | Type        | Description                                                           |
 | --------------- | ----------- | --------------------------------------------------------------------- |
-| `fxa_uid`       | `UUID`      | Firefox Account UID PK (part 1)                              |
+| `fxa_uid`       | `BIGINT`      | Firefox Account UID PK (part 1)                              |
 | `fxa_kid`       | `TEXT`      | Key identifier; part of the sync crypto context. PK (part 2) |
 | `collection_id` | `INTEGER`    | Maps to a named collection. PK (part 3)                       |
-| `modified`      | `TIMESTAMP` | Last modification time (server-assigned, updated on writes)           |
+| `modified`      | `BIGINT` | Last modification time (server-assigned, updated on writes)           |
```

Comment:
Is this too bothersome to support as a `TIMESTAMP`?

Reply:
Not necessarily, may opt for a raw query over the ORM methods, as we've discussed, given those can be a pain with the chain methods. Do we prefer this to be a timestamp? I know in Spanner it's a timestamp and in MySQL a BIGINT

Reply:
I think I understand your explanation, @taddes, thanks.  But I don't agree or maybe simply don't understand why the existing tests should stop us from using the correct data type(s).

> My thought here is that we try and keep true-ish to MySQL and as we near the end of integrating the tests, look at making adjustments if need be. 

How do we decide if adjustments are needed?